css/css-backgrounds/border-image-space-001.html

```diff
@@ -6,7 +6,7 @@
   <link rel="author" title="Levi Weintraub" href="mailto:leviw@chromium.org">
   <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-repeat">
   <meta name="assert" content="border-image-repeat: space property spaces out background image that doesn't fit an even number of times.">
-  <meta name="fuzzy" content="maxDifference=0-80;totalPixels=0-1728">
+  <meta name="fuzzy" content="maxDifference=0-255; totalPixels=0-3895">
```

Comment:
This is suspicious, it's suggesting 3895 pixels can have completely different color (0-255 is the max range).

Can we revert https://commits.webkit.org/262031@main and this change from the PR?

Reply:
Yes -- I'll do a separate change for 262031@main in WebKit, and remove this change here.